.github/workflows/build_image.yml

```diff
@@ -9,6 +9,7 @@ jobs:
       uses: actions/checkout@main
 
     - name: update jupyter dependencies with repo2docker
-      uses: jupyterhub/repo2docker-action@6a93d1bf173f32cfd859130d5851aa8a7aad07cd  # master as of 2020-10-20
+      uses: NSLS-II/repo2docker-action@c64bda966539ff36856055311be7d709605a8a97  # adds support for --appendix
```

Comment:
Nice addition! https://github.com/jupyterhub/repo2docker-action/pull/72 is for future reference.